docker/docker-compose.yml

```diff
@@ -6,12 +6,14 @@ services:
       context: ..
       dockerfile: docker/Dockerfile
       args:
+        env: "production"
         dev: "--no-dev"
         simulate_removal: ""
     image: libero/search:${IMAGE_TAG:-master}
     command: uwsgi --ini=uwsgi.ini
     volumes:
       - ./uwsgi.ini:/srv/app/uwsgi.ini
+      - ./constants.dev.py:/srv/app/search/constants.py
```

Comment:
Needs an Elasticsearch running too?

Reply:
Short answer is yes. This service is merely an interface for Elasticsearch that performs search related tasks.

Regarding line 16, this was added to pass required settings to the application during CI smoke tests.

Reply:
Ok. Elsewhere we have `docker-compose.yml` as a way to bring up the project locally in a `prod`-like environment, sounds like this is for something else.